tests/flow/test_unwind_clause.py

```diff
@@ -0,0 +1,88 @@
+from common import *
+import re
+
+redis_graph = None
+
+class testUnwindClause(FlowTestsBase):
+    
+    def __init__(self):
+        self.env = Env(decodeResponses=True)
+        global redis_graph
+        redis_con = self.env.getConnection()
+        redis_graph = Graph(redis_con, "G")
```

Comment:
```python
GRAPH_ID = "unwind"
...
 redis_graph = Graph(redis_con, GRAPH_ID)
```

Reply:
Ok, change done in [d68e6b1](https://github.com/RedisGraph/RedisGraph/pull/2650/commits/d68e6b176ba7b549700390b4ad0a81d2ad9fef2d). Thanks